packages/devtools_app/lib/src/screens/performance/timeline_flame_chart.dart

```diff
@@ -434,26 +440,26 @@ class TimelineFlameChartState
     // improve performance.
     if (_selectedFrame != null) {
       // Zoom and scroll to the frame's UI event.
-      final time = _selectedFrame.timeToCenterFrame();
-      final event = _selectedFrame.eventToCenterFrame();
+      final time = _selectedFrame!.timeToCenterFrame();
```

Comment:
Hoist `_selectedFrame` into a local above the condition above.

Reply:
done